src/css/index.scss

```diff
@@ -0,0 +1,418 @@
+$primary_color: #ddd;
+$secondary_color: #fff;
+$btn_border_color: red;
+$btn_hover_color: #ff7675;
+
+* {
+  box-sizing: border-box;
+  font-family: 'montserrat', sans-serif;
+  background: transparent;
+  .slider {
+    display: none;
+  }
+}
+
+.bgimg {
+  background: url('../css/img/cv.jpg');
+  background-size: cover;
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: -1;
+  filter: blur(5px);
+}
+
+.header {
+  max-width: 1000px;
+  width: 100%;
+  padding: 0 auto;
+  height: 100%;
+  margin: auto;
+  a:nth-child(1) {
+    position: relative;
+    left: 0;
+    max-width: 50%;
+    text-decoration: none;
+    font-size: 24px;
+    color: $secondary_color;
+    margin: 0 25px;
+    font-weight: bold;
+  }
+  a:nth-child(2) {
+    display: flex;
+    margin: 0 25px;
+    flex-wrap: wrap;
+    position: relative;
+    max-width: 50%;
+    right: 0;
+    float: right;
+    text-decoration: none;
+    border: 1px solid $btn_border_color;
+    font-size: 18px;
+    padding: 4px 25px;
```

Comment:
To make the button/single-line content to be vertical centre, we normally set button's line-height equal to its height.

```css
height: 30px;
line-height: 30px;
```